wp-content/themes/access/views/home.twig

```diff
@@ -55,34 +55,36 @@
 
             <div class="screen-tablet:layout-columns-gutter">
               {% for tout in homepage_touts %}
-              <div itemscope itemprop="{{ tout.item_scope }}" itemtype="https://schema.org/{{tout.item_scope}}" class="c-card p-0 pb-4">
+              <div {% if tout.item_scope %}itemscope itemprop="{{ tout.item_scope }}" itemtype="https://schema.org/{{tout.item_scope}}"{% endif %} class="c-card p-0 pb-4">
                 <div class="c-card__body">
                   {% if tout.tout_title %}
-                  <h3 itemprop="name" class="c-card__title mt-0">
+                  <h3 {% if tout.item_scope %}itemprop="name"{% endif %} class="c-card__title mt-0">
                     {% if tout.status %}
                     <mark class="badge color-{{ tout.status.type }}-status">{{ tout.status.text }}</mark>
                     {% endif %}
 
-                    <a itemprop="newsUpdatesAndGuidelines" class="text-blue-dark" href="{{ tout.link_url }}"{% if tout.link_new_window %} target="_blank" rel="noopener noreferrer"{% endif %}>
+                    <a {% if tout.item_scope %}itemprop="newsUpdatesAndGuidelines"{% endif %} class="text-blue-dark" href="{{ tout.link_url }}"{% if tout.link_new_window %} target="_blank" rel="noopener noreferrer"{% endif %}>
                       {{ tout.tout_title }}
                     </a>
                   </h3>
                   {% endif %}
 
                   <p class='c-card__subtitle color__alt'>
                     {{ __('Updated', 'accessnyctheme') }} {{ tout.post_modified|date('l, F jS, g:ia') }}
-                    <meta itemprop="datePosted" content="{{ tout.post_modified }}" />
+                    {% if tout.item_scope %}
+                      <meta itemprop="datePosted" content="{{ tout.post_modified }}" />
+                    {% endif %}
                   </p>
 
                   {% if tout.link_text %}
-                    <div itemprop="text" class="c-card__summary">{{ tout.link_text }}</div>
+                    <div {% if tout.item_scope %}itemprop="text"{% endif %} class="c-card__summary">{{ tout.link_text }}</div>
                   {% endif %}
 
-                  {% if tout.item_scope == 'SpecialAnnouncement' %}
+                  {% if tout.item_scope %}
                     <meta itemprop="category" content="https://www.wikidata.org/wiki/Q81068910" />
                   {% endif %}
 
-                  {% if tout.custom.tout_status_clear_date %}
+                  {% if tout.custom.tout_status_clear_date and tout.item_scope %}
                     <meta itemprop="expires" content="{{ tout.custom.tout_status_clear_date }}" />
```

Comment:
@devowhippit here I added the conditional `and tout.item_scope` to make sure the ` tout.custom.tout_status_clear_date` is not nil and is included. I can just make it `{% if tout.item_scope %}` if you like.